components/expenses/CreateExpenseForm.js

```diff
@@ -438,17 +447,23 @@ class CreateExpenseForm extends React.Component {
 
             <div className="col">
               <label>
-                <FormattedMessage id="expense.type" defaultMessage="type" />
+                <FormattedMessage
+                  id="ExpenseType"
+                  defaultMessage="{ExpenseType}"
```

Comment:
There's no need for a `FormattedMessage` if it just renders the parameter. You can directly put `<DefinedTerm term={Terms.EXPENSE_TYPE} />`